internal/cmd/run/run.go

```diff
@@ -108,6 +108,7 @@ func Command() *cobra.Command {
 	cmd.PersistentFlags().DurationVarP(&gFlags.globalTimeout, "timeout", "t", 0, "Global timeout that limits how long saucectl can run in total. Supports duration values like '10s', '30m' etc. (default: no timeout)")
 	cmd.PersistentFlags().BoolVar(&gFlags.async, "async", false, "Launches tests without waiting for test results (sauce mode only)")
 	cmd.PersistentFlags().BoolVar(&gFlags.failFast, "fail-fast", false, "Stops suites after the first failure (sauce mode only)")
+	cmd.PersistentFlags().DurationVar(&gFlags.appStoreTimeout, "uploadTimeout", 300*time.Second, "Upload timeout that limits how long saucectl can upload test files. Supports duration values like '10s' '30m' etc. (default: 300 seconds)")
```

Comment:
Might as well make the values a bit more human friendly :D

```suggestion
	cmd.PersistentFlags().DurationVar(&gFlags.appStoreTimeout, "uploadTimeout", 5*time.Minute, "Upload timeout that limits how long saucectl will wait for an upload to finish. Supports duration values like '10s' '30m' etc. (default: 5m)")
```